app/controllers/items_controller.rb

```diff
@@ -14,7 +15,6 @@ def index
 
   def show
     # show a specific item on click
-    @item = Item.find(params[:id])
```

Comment:
are you removing the show function?

Reply:
No, I create a private find method that is called in all the methods. Just refactoring the code

Reply:
def find_item
    @item = Item.find(params[:id])
  end

instead of running @item= Item.find... in every method